src/worker/cluster-worker.ts

```diff
@@ -0,0 +1,98 @@
+import { AsyncResource } from 'async_hooks'
```

Comment:
async_hooks should not be needed for cluster, that is intended to be used for worker_threads for what I remember

Reply:
If I remove this `async_hooks` I get errors for non exisiting function `runInAsyncScope` and `emitDestroy`
I don't even know what `async_hooks` is, I just copied over the thread stuff

Reply:
https://nodejs.org/dist/latest-v14.x/docs/api/async_hooks.html

Is a node js module and an asynchronous resource represents an object with an associated callback, so maybe we can stay with it